### PR TITLE
Fix #244 bacteriophages treated as viruses

### LINKED ISSUE #244: Bacteriophage should be marked as virus and not parsed
Bacteriophage Cf1c
Bacteriophage PH75
Bacteriophage PH75
Bacteriophage PH75

### PR DESCRIPTION
**Pull request by dimus**

### CODE REVIEW

File: parser/src/main/scala/org/globalnames/parser/Preprocessor.scala [hunk 1]
```diff
@@ -63,9 +63,8 @@ object Preprocessor {
 
   private final val virusPatterns =
     """\sICTV\s*$""".r :: """[A-Z]?[a-z]+virus\b""".r ::
-      """(?ix)\b(virus|viruses|particle|particles|
-               phage|phages|viroid|viroids|virophage|
-               prion|prions|NPV)\b""".r ::
+      """(?ix)\b(virus(es)?|particle(s)?|(bacterio|viro)?phage(s)?|
```
Comment (alexander-myltsev) [inline parser/src/main/scala/org/globalnames/parser/Preprocessor.scala]: Why do you need to put single character `s` to parentheses? Regex engine creates a group for it, i.e. consumes more CPU cycles and memory.

File: parser/src/test/resources/test_data.txt [hunk 2]
```diff
@@ -1558,6 +1558,18 @@ Zamilon virophage
 Sputnik virophage 3
 {"name_string_id":"b206bb35-01bf-59a7-8dad-bc8f99ca0a2a","parsed":false,"parser_version":"test_version","verbatim":"Sputnik virophage 3","surrogate":false,"virus":true,"bacteria":false}
 b206bb35-01bf-59a7-8dad-bc8f99ca0a2a|Sputnik virophage 3|||||1
+
+Bacteriophage PH75
+{"name_string_id":"605f428e-a4a3-57a2-9dfa-a6a3d99b801d","parsed":false,"parser_version":"test_version","verbatim":"Bacteriophage PH75","surrogate":false,"virus":true,"bacteria":false}
+605f428e-a4a3-57a2-9dfa-a6a3d99b801d|Bacteriophage PH75|||||1
+
+Bacteriophage PH75
```
Comment (dimus) [inline parser/src/test/resources/test_data.txt]: yep, removing...

File: parser/src/main/scala/org/globalnames/parser/Preprocessor.scala (hunk 1 above)
Comment (dimus) [inline parser/src/main/scala/org/globalnames/parser/Preprocessor.scala]: good point

**Comment by alexander-myltsev** [comment]
LGTM

**Pull request merged**